mindmaps-core/src/main/java/io/mindmaps/core/implementation/MindmapsTransactionImpl.java

```diff
@@ -44,23 +44,17 @@
     private final ElementFactory elementFactory;
     private final ConceptLog conceptLog;
     private final AbstractMindmapsGraph mindmapsGraph;
-    private boolean batchLoadingEnabled;
     private Graph transaction;
 
     public MindmapsTransactionImpl(AbstractMindmapsGraph mindmapsGraph) {
         this.mindmapsGraph = mindmapsGraph;
         this.transaction = mindmapsGraph.getGraph();
-        this.batchLoadingEnabled = false;
         conceptLog = new ConceptLog();
         elementFactory = new ElementFactory(this);
     }
 
-    public void setBatchLoadingEnabled(boolean batchLoadingEnabled){
-        this.batchLoadingEnabled = batchLoadingEnabled;
-    }
-
     public boolean isBatchLoadingEnabled(){
-        return batchLoadingEnabled;
+        return mindmapsGraph.isBatchLoadingEnabled();
```

Comment:
I'm not sure. I think the accessor for a graph might be a good thing in the long run - I've needed it from time to time, but I see your point too.

More importantly, it sounds like maybe we'll be merging transaction and graph back again, so it's probably not worth worrying about!